system/jlib/jsocket.cpp

```diff
@@ -105,6 +105,10 @@
 #define _TRACE
 #endif
 
+#if 1 // spurious mplink closed tracing info
+# define _TRACELINKCLOSED
+#endif
+
```

Comment:
minor: I added same in mpcomm.cpp (in gh-9281). Would have been moved that into jsocket.hpp